src/components/20-molecules/policy-features/policy-features-item/ui.test.js

```diff
@@ -0,0 +1,115 @@
+
+fixture('Policy features item - basic functionality').page(
+  `${host}/iframe.html?id=molecules-policy-features--policy-features`
+);
+
+test('should render policy-features-item', async t => {
+  const $axaElem = await Selector(TAG);
+  await t.expect($axaElem.exists).ok();
+  const $axaElemShadow = await Selector(
+    () => document.querySelector('axa-policy-features-item').shadowRoot
+  );
+  const $axaElemShadowEl = await $axaElemShadow.find(CLASS);
+  await t.expect($axaElemShadowEl.exists).ok();
+});
+
+test('should render title of first item', async t => {
+  await t.expect(firstH1ElementInDocument.textContent).eql('Get Discount');
+});
+
+test('should render description of first item', async t => {
+  await t
+    .expect(firstPElementInDocument.textContent)
+    .eql('A 5 star car insurance with affordable premium services');
+});
+
+test('should render svg icon of first item', async t => {
+  await t.expect(firstSvgElementInDocument.exists).ok();
+});
+
+test('should set svg size for screen size md-up', async t => {
+  await t
+    .expect(await firstSvgElementInDocument.getStyleProperty('width'))
+    .eql('96px')
+    .expect(await firstSvgElementInDocument.getStyleProperty('height'))
+    .eql('96px');
+}).before(async t => {
+  await t.resizeWindow(mdWindowWidth, defaultWindowHeight);
+});
+
+test('should set svg size for screen size smaller than md', async t => {
+  await t
+    .expect(await firstSvgElementInDocument.getStyleProperty('width'))
+    .eql('42px')
+    .expect(await firstSvgElementInDocument.getStyleProperty('height'))
+    .eql('42px');
+}).before(async t => {
+  await t.resizeWindow(smWindowWidth, defaultWindowHeight);
+});
+
+test('should render title and description with size medium-1 on screen md', async t => {
+  await t
+    .expect(await firstH1ElementInDocument.getStyleProperty('font-size'))
+    .eql('18px')
+    .expect(await firstPElementInDocument.getStyleProperty('font-size'))
+    .eql('18px');
+}).before(async t => {
+  await t.resizeWindow(mdWindowWidth, defaultWindowHeight);
+});
+
+test('should render title and description with size medium on screen size smaller than md', async t => {
+  await t
+    .expect(await firstH1ElementInDocument.getStyleProperty('font-size'))
+    .eql('16px')
+    .expect(await firstPElementInDocument.getStyleProperty('font-size'))
+    .eql('16px');
+}).before(async t => {
+  await t.resizeWindow(smWindowWidth, defaultWindowHeight);
+});
+
+test('should set width of item with screen size lg-up', async t => {
+  await t
+    .expect(await firstArticleElementInDocument.getStyleProperty('width'))
+    .eql('255px');
+}).before(async t => {
+  await t.resizeWindow(lgWindowWidth, defaultWindowHeight);
+});
+
+test('should set width of item with screen size md-up', async t => {
+  await t
+    .expect(await firstArticleElementInDocument.getStyleProperty('width'))
+    .eql('210px');
+}).before(async t => {
+  await t.resizeWindow(mdWindowWidth, defaultWindowHeight);
```

Comment:
if u want u can add this before on fixure level so that u dont have to copy it on every test like so: https://github.com/axa-ch/patterns-library/blob/1f5565ed8e264ff1e81819e2c417cdeb50534868/src/components/30-organisms/table-sortable/ui.test.js#L10

Reply:
Every test needs another screen size - so i have to write it that way